bump_rpm.sh

```diff
@@ -30,7 +30,7 @@ ensure_program() {
 }
 
 if [[ -z $2 ]] ; then
-	if [[ $PACKAGE_NAME == rubygem-* ]] || [[ $PACKAGE_NAME == tfm-rubygem-* ]]; then
+	if echo $PACKAGE_NAME | grep -q rubygem; then
```

Comment:
You could also keep it pure bash by changing `tfm-rubygem-*` to `*-rubygem-*`.

Reply:
Good idea, I've also noticed we have this condition in some other location, so fixed it there as well